### ISSUE TITLE
Spectral Density Calculation    

### ISSUE THREAD

**Issue by piskuliche** [piskuliche/Spectroscopy#10]
It would be helpful to eventually add the spectral density calculations for SFG and Raman to the calculation.

Short Term:

- [x] Add SFG Spectral Density pol-dipole
- [x] Add Raman Spectral Density pol-pol

Longer Term:

- [ ] Add frequency distribution at the interface

**Comment by piskuliche** [comment]
I've made significant progress on this today. 

New features: 
-> SFG spectral density added
-> Raman spectral density added

Also - added the two dimensional histograms ofr spectral density and frequency distribution for the SFG calculation based on zdistance.

Currently bug testing, but should be in production soon.

It should be noted that this does include a change to the input file format, to add a line zmin, zmax, zhist after the w1min, w1max, ... line. This is to make the 2D histogram functional. Also, the deprecated ndigits option was removed.

- [x] Test Bugs
- [x] Frequency distribution att the interface
